#manipulator geometry parameters:
base_radius:               0.09
platform_radius:           0.05
base_joint_spacing:        0.0375
platform_joint_spacing:    0.0375
proximal_link_length:      0.12
distal_link_length:        0.27069
retracted_position:        [0.0, 0.0, 0.18]
home_offset:               [0.0, 0.0, 0.03]
translation_limit:         0.12

#dont change this
rotation_limit:            0.0